### PR TITLE
MongoDB and Daily Entry Schema

### PR DESCRIPTION
**Pull request by RetroInferno**
MongoDB now connected and should be passing the automated tests. Please follow instructions in the ReadMe to set up your local  `.env`. URI for mongoDB will be sent through Slack

### CODE REVIEW

File: server/db/connect.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      });
+      // console.log(`Connected to MongoDB Atlas, Database ${database}`);
+      return connection;
+    } catch (err) {
+      console.log('Failed to connect to MongoDB Atlas')
+      console.log(err);
+      throw err;
+    }
+  }
+}
```
Comment (slava000) [inline server/db/connect.ts:44]: can we keep this DRY? I.e. one mongoConnection expression, and having conditional logic on the const connection.

File: server/db/schema.models.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import mongoose from 'mongoose';
+const dailyEntriesName = 'daily_entries';
+
```
Comment (slava000) [inline server/db/schema.models.ts:3]: what's the status / plan for the users and goals models?

**Review by slava000 — APPROVED** [review]

File: server/db/connect.ts (hunk 1 above)
Comment (RetroInferno) [reply server/db/connect.ts:44]: Yeah I for sure! We can get it on the next revision! This was a bit rough cause I was just trying to get the githubActions to connect to the DB using SCRAM.

File: server/db/schema.models.ts (hunk 2 above)
Comment (RetroInferno) [reply server/db/schema.models.ts:3]: That should be done or decided by the owners of the Authentication and users features. Essentially all we care about is the user_id String, which can be anything

**Pull request merged**